templates/producer.init.erb

```diff
@@ -21,7 +21,7 @@ start() {
     else
         exec $DAEMON $DAEMON_OPTS >> /var/log/kafka/producer.log 2>&1 &
         sleep 2
-        PID=`ps axww | grep -E '[k]afka-console-producer' | awk '{print $1}'`
+        PID=`ps axww | grep -E '[k]afka.tools.ConsoleProducer' | awk '{print $1}'`
```

Comment:
Nothing and I have no idea if the regular expression is necessary. 
BTW: You missed ```init.erb``` und ```mirror.init.erb``` ;)